### ISSUE TITLE
NTR: Bile alcohols high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8270]
New term request Bile alcohols high in blood (elthat_101120220842):

**Comment by pnrobinson** [comment]
@MickeySegal
I am not very familiar with this, but it seems that specific alcohols were measured and there may not be a test for total bile alcohol that is in use today
see https://pubmed.ncbi.nlm.nih.gov/3067328/

**Comment by MickeySegal** [comment]
Table 1 in https://www.ncbi.nlm.nih.gov/books/NBK1409/ makes it sound like testing is done clinically.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000821
- label: Elevated circulating bile alcohol concentration
- parents: [Abnormal circulating organic compound concentration (HP:0430071)](https://hpo.jax.org/app/browse/term/HP:0430071)
- pmid: PMID:20301583
- def: The concentration of bile alcohols in the blood circulation is above the upper limit of normal.
- comment: This phenotypic feature may be observed in Cerebrotendinous Xanthomatosis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8270
- synonyms:
